source/menus/cutscene_flag_menu.cpp

```diff
@@ -3,14 +3,17 @@
 #include "libtww/d/com/d_com_inf_game.h"
 #include "flags.h"
 
-#define LINE_NUM 2
+#define LINE_NUM 3
 Cursor CutsceneFlagMenu::cursor;
 int cursorOffset;
 Line lines[LINE_NUM] = {
     {"Watched Orca Intro Cutscene", ORCA_INTRO_CUTSCENE, "Sets initial Orca cutscene to seen", true,
         &g_flags[ORCA_INTRO_CUTSCENE]},
     {"Watched Puppet Ganon Cutscene", PUPPET_GANON_CUTSCENE, "Sets Puppet Ganon cutscene to seen", true,
      &g_flags[PUPPET_GANON_CUTSCENE]},
+    {"Courtyard Cutscene", COURTYARD_CUTSCENE, "Sets H1 courtyard cutscene to seen", true,
```

Comment:
I prefer spelling out acronyms and abbreviations for the sake of newer runners ("H1" --> "Hyrule 1")